<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nim::NimBuildConfigurationWidget</class>
 <widget class="QWidget" name="Nim::NimBuildConfigurationWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="targetLabel">
       <property name="text">
        <string>Target:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="targetComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="defaultArgumentsLabel">
       <property name="text">
        <string>Default arguments:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="defaultArgumentsComboBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Release</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>targetComboBox</tabstop>
  <tabstop>defaultArgumentsComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
